syntax = "proto3";

service OrderService {
  rpc CalculateTotal(OrderRequest) returns (OrderResponse) {}
}

message OrderRequest {
  string product_id = 1;
  int32 quantity = 2;
}

message OrderResponse {
  float total = 1;
  string status = 2;
}